src/frontend/js/index.js

```diff
@@ -334,7 +336,7 @@ const mapboxInit = ({lng, lat, zoom}) => {
             },
             paint: {
                 'line-opacity': .8,
-                'line-color': 'red',
+                'line-color': weightToColor(1),
```

Comment:
This whole section can go, which then means we can put weightToColor in paintCentroids as a closure, which I think is better (no confusion about what it is for or where it is used).

Reply:
Agreed & done.